ods_ci/utils/scripts/testconfig/generateTestConfigFile.py

```diff
@@ -174,8 +174,8 @@ def get_dashboard_url():
     """
     Get dashboard url for the open data science.
     """
-    cmd = "oc get route -A -o json  | jq '.items[].spec.host' | grep 'dashboard'"
-
+    cmd = "(oc get route -n opendatahub-operators -o json  | jq '.items[].spec.host' | grep odh-dashboard) || " + \
+          "(oc get route -n redhat-ods-applications -o json  | jq '.items[].spec.host' | grep rhods-dashboard)"
```

Comment:
`redhat-ods-applications` - will this work for ODH? :thinking: 
`rhods-dashboard` - this definitely won't work for ODH